src/Microsoft.Health.Fhir.Liquid.Converter.FunctionalTests/TestData/Expected/Hl7v2/ADT_A01/ADT01-23-expected.json

```diff
@@ -196,6 +196,12 @@
               "reference": "Location/"
             },
             "status": "active"
+          },
+          {
+            "location": {
+              "reference": "Location/9b2956e2-230b-9dd5-ad8d-0e697a8664c1"
+            },
```

Comment:
It seems we should replace the original one instead of adding a new one.

Reply:
Yes, the blank reference is caused by a covert issue(the wrong condition of resource templates), currently it has been fixed.